packages/lib/services/plugins/pluginCategories.ts

```diff
@@ -0,0 +1,64 @@
+export default function pluginCategories() {
+type pluginCategory = {
+	[key: string]: string[];
+};
+const pluginCategoriesOptions: pluginCategory = {
```

Comment:
we don't do that I think. That should be up to the plugin developers to set the category

Reply:
Yes. It's only for the demo purpose. I will be removing it before merging.

Reply:
ok but in the meantime we can't merge

Reply:
Okay then I will remove it.